packages/react/src/blocks/Symbol.tsx

```diff
@@ -135,7 +136,8 @@ class SymbolComponent extends React.Component<SymbolProps> {
                   renderLink={state.renderLink}
                   inlineContent={symbol?.inline}
                   {...(content && { content })}
-                  options={{ key, noEditorUpdates: true }}
+                  key={builderComponentKey}
+                  options={{ key: builderComponentKey, noEditorUpdates: true }}
```

Comment:
Ah, so on change, because the key was not changing, react was not updating? Great find!

Reply:
Nice, setting a `key` will force React to re-render the component completely if the `key` changes.

Since the `key` is generated based on the `locale`, `model` and `entry` values, it should be safe to do that 👍🏽